app/controllers/media_controller.rb

```diff
@@ -47,6 +49,17 @@ def auth_check
 
   private
 
+  # We do not rely on the web server to set Access-Control-Allow-Origin for *any* /media request,
+  # so we set it manually ourselves.
+  def set_origin_header
+    response.headers['Access-Control-Allow-Origin'] = '*'
+  end
+
+  def set_cors_headers
+    response.headers['Access-Control-Allow-Origin'] = Settings.cors.allow_origin_url
+    response.headers['Access-Control-Allow-Credentials'] = Settings.cors.allow_credentials
```

Comment:
I can't think of the case where we would ever want to set this to anything other than `true`, so I wonder if this is a good use of a configuration.  It's not particularly harmful, but I would be concerned that somebody might take this to indicate there is a use for setting this to `false`.  Up to you.